apps/etl/tasks.py

```diff
@@ -0,0 +1,273 @@
+import hashlib
+import json
+import logging
+import typing
+from datetime import datetime, timedelta
+
+import numpy as np
+import pandas as pd
+from celery import shared_task
+from django.core.files.base import ContentFile
+from django.core.management import call_command
+from pydantic import ValidationError
+
+from apps.etl.extract import Extraction
+from apps.etl.extraction_validators.gdacs_events_validator import (
+    GdacsEventsDataValidator,
+)
+from apps.etl.extraction_validators.gdacs_eventsdata_geometry_validator import (
+    GdacsEventsGeometryData,
```

Comment:
This is all looking really good! One question I had is the relationship between tasks and sources. Are all sources going to be handled in a single task? Asking because this tasks.py looks like it's only dealing with GDACs but there's no abstraction to add another one?